test/sales_engine_test.rb

```diff
@@ -29,4 +29,19 @@ def test_we_can_match_items_to_merchant
     expected = [merchant]
     assert_equal 1, expected.count
   end
+
+  def test_we_can_get_all_items_by_merchant_id
+    sales_engine = SalesEngine.from_csv({
+    :items     => "./fixtures/items_sample.csv",
+    :merchants => "./fixtures/merchant_sample.csv",
+    })
+
+    mr = sales_engine.merchants
+    # require "pry"; binding.pry
+    found_items = sales_engine.find_items_by_id(12334195)
+    #an array of items in found_items
```

Comment:
I think this would good for getting our thoughts across when visualizing what we needed to return when building out our test, but we can take this out as well now that we have our test functioning.

Reply:
got it! Good catch @klatour324 !